src/bundle/Command/UpdateUserCommand.php

```diff
@@ -0,0 +1,119 @@
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\User\Command;
+
+use Ibexa\Contracts\Core\Repository\Repository;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Repository\Values\User\User;
+use Ibexa\Contracts\Core\Repository\Values\User\UserUpdateStruct;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(name: 'ibexa:user:update-user', description: 'Updates basic user data.')]
+final class UpdateUserCommand extends Command
+{
+    public function __construct(
+        private readonly UserService $userService,
+        private readonly Repository $repository,
+        ?string $name = null
+    ) {
+        parent::__construct($name);
+    }
+
+    protected function configure(): void
+    {
+        $this->addArgument(
+            'user',
+            InputArgument::REQUIRED,
+            'User reference (id or login)',
+        );
+        $this->addOption(
+            'password',
+            null,
+            InputOption::VALUE_NONE,
+            'New plaintext password (type will be in a "hidden" mode)',
+        );
+        $this->addOption(
+            'email',
+            null,
+            InputOption::VALUE_REQUIRED,
+            'New e-mail address',
+        );
+        $this->addOption(
+            'enable',
+            null,
+            InputOption::VALUE_NONE,
+            'Flag enabling the user being updated',
+        );
+        $this->addOption(
+            'disable',
+            null,
+            InputOption::VALUE_NONE,
+            'Flag disabling the user being updated',
+        );
+    }
+
+    /**
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $userReference = $input->getArgument('user');
+        $password = $input->getOption('password');
+        $enable = $input->getOption('enable');
+        $disable = $input->getOption('disable');
+        $email = $input->getOption('email');
+
+        if (!$password && !$enable && !$disable && $email === null) {
+            $io->success('No new user data specified, exiting.');
+
+            return Command::SUCCESS;
+        }
+
+        if (is_numeric($userReference)) {
+            $user = $this->userService->loadUser((int)$userReference);
+        } else {
+            $user = $this->userService->loadUserByLogin($userReference);
+        }
```

Comment:
What if user login is numeric? Or does our validation forbid such case?

Not a huge fan of this condition. We should rather stick to login only. Let's not advocate solutions which strongly rely on database-generated IDs.